sys/dev/athn/athn.c

```diff
@@ -667,6 +669,7 @@ athn_reset_power_on(struct athn_softc *sc)
 {
 	int ntries;
 
+	ATHN_LOCK(sc);
```

Comment:
I see there is: `DELAY(2);` inside couple lines below. Won't it crash?

Reply:
athn_reset_power_on returns 0, it seems it pass